lib/docker-shell-command-executor.ts

```diff
@@ -32,7 +33,10 @@ export default class DockerShellCommandExecutor {
   }
 
   async exec(command: string) {
-    await exec(`docker run --rm -v ${this.workingDirectory}:/workdir -w /workdir course-sdk-${this.dockerfileType} ${command}`);
+    let quotedCommand = command.replace(/\\/g, '\\\\')
+            .replace(/"/g, '\\"')
+            .replace(/'/g, "\\'");
```

Comment:
@ryan-gang why do single quotes need to be replaced when we're surrounding a string with double quotes?